2021/fall/Sterne-TTh@10/yuehhan-huang.md

```diff
@@ -0,0 +1,84 @@
+## Yueh Han Huang
+
+
+### What I have been reading recently 
+
+- [Curius.app](https://curius.app/yueh%20han-huang)
+
+
+### M22  Cities 
+- San Frnacisco 
+- Seoul 
+- Hyderabad 
+- Berlin 
+- Buenos Aires
+- London 
+- Taipei （rip)
+
+
+### Music 
+
+[Study Music](https://open.spotify.com/playlist/0F9ihuHVcbUds4QG9asqsQ?si=6628c829718848a8)
+
+### Some links 
+
+<details>
+    <summary>My Digital Presence - elsewhere</summary>
+    <ul> 
+        <li>
+            <a href="https://are.na/yueh-han-huang" >are.na/yueh-han-huang</a> 
+        </li>
+        <li>
+            <a href="https://vsco.co/yuehan18" >vsco.co/yuehan18</a> 
```

Comment:
@Bojne , when I clink on the link, the whole page reloads. Is there a way to click on the link and be sent to a new tab, so your profile does not have to be reloaded? It might be helpful to add this to your anchor tag: target="_blank"